src/target/test/pnglite.c

```diff
@@ -0,0 +1,877 @@
+/*  pnglite.c - pnglite library
+    For conditions of distribution and use, see copyright notice in pnglite.h
+*/
+#define DO_CRC_CHECKS 1
+#define USE_ZLIB 1
+
+#if USE_ZLIB
+#include "zlib.h"
+#else
+#include "zlite.h"
+#endif
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include "pnglite.h"
+
+
+
+static png_alloc_t png_alloc;
+static png_free_t png_free;
+
+static size_t file_read(png_t* png, void* out, size_t size, size_t numel)
+{
+	size_t result;
+	if(png->read_fun)
+	{
+		result = png->read_fun(out, size, numel, png->user_pointer);
+	}
+	else
+	{
+		if(!out)
+		{
+			result = fseek(png->user_pointer, (long)(size*numel), SEEK_CUR);
```

Comment:
Lint Error:
```
30:  Tab found; better to use spaces  
31:  Tab found; better to use spaces  
31:  Missing space before {  
32:  Tab found; better to use spaces  
33:  Tab found; better to use spaces  
33:  Missing space before {  
34:  Tab found; better to use spaces  
34:  Use int16/int64/etc, rather than the C type long  
```